src/LibraryManager/Manifest.cs

```diff
@@ -124,35 +140,34 @@ public void ReplaceLibraryId(ILibraryInstallationState libraryToUpdate, string n
             }
         }
 
-        private static void UpdateLibraryProviderAndDestination(Manifest manifest)
-        {
-            foreach (LibraryInstallationState state in manifest.Libraries.Cast<LibraryInstallationState>())
-            {
-                UpdateLibraryProviderAndDestination(state, manifest.DefaultProvider, manifest.DefaultDestination);
-            }
-        }
-
-        private static void UpdateLibraryProviderAndDestination(ILibraryInstallationState state, string defaultProvider, string defaultDestination)
-        {
-            LibraryInstallationState libraryState = state as LibraryInstallationState;
-
-            if (libraryState == null)
-            {
-                return;
-            }
-
-            if (state.ProviderId == null)
-            {
-                libraryState.ProviderId = defaultProvider;
-                libraryState.IsUsingDefaultProvider = true;
-            }
-
-            if (libraryState.DestinationPath == null)
-            {
-                libraryState.DestinationPath = defaultDestination;
-                libraryState.IsUsingDefaultDestination = true;
-            }
-        }
+        //private static void UpdateLibraryProviderAndDestination(Manifest manifest)
```

Comment:
>//private static void UpdateLibraryProviderAndDestination(Manifest manifest) [](start = 8, length = 76)

I assume you want this commented out rather than deleted?